PCAxis.Excel/XlsxSerializer.cs

```diff
@@ -175,25 +175,12 @@ private XLWorkbook CreateWorkbook(PCAxis.Paxiom.PXModel model)
 						row = 3 + model.Meta.Heading.Count + i;
 						column = j * dataNoteFactor + sIndent + 1;
 						value = fmt.ReadElement(i, j, ref n, ref dataNote);
-                        //*Decimal valueDecimal = Decimal.Parse(value.ToString());
-                        //*string valueDecimal = "70,0";
-                        //if (!value.IsNumeric())
-                        //{
-                        //    sheet.Cell(row, column + dataNoteValueOffset).Style.Alignment.Horizontal = XLAlignmentHorizontalValues.Right;
-                        //    sheet.Cell(row, column + dataNoteValueOffset).Style.Fill.BackgroundColor = XLColor.LightSalmon;
-                        //}
-                        //else
-                        //{
-                        //    sheet.Cell(row, column + dataNoteValueOffset).DataType = XLCellValues.Number;
-                        //}
-                        //sheet.Cell(row, column + dataNoteValueOffset).Value = value;
-
-                        //2019-01-09 Removed background Color color in cell, lightsalmon. Do not add background color when cell contains a non numeric value.
-                      
-                        setCell(
-                            sheet.Cell(row, column + dataNoteValueOffset),                           
-                            CellContentType.Data,
-                            value,
+
+						//TODO: Improve performance of setting value format, takes a lot of CPU at the moment
+						setCell(
+							sheet.Cell(row, column + dataNoteValueOffset),
+							CellContentType.Data,
+							value,
                             !value.IsNumeric() ?
```

Comment:
Is it line 200 that has the performance problem?

Reply:
It is row 184